libraries/core-react/package.json

```diff
@@ -27,7 +27,6 @@
     "dist/*"
   ],
   "scripts": {
-    "prepare": "npm run build",
```

Comment:
suggest we change this to `pnpm` instead of removing altogether to avoid human errors when publishing to npm (god, we need to fix publishing this with pipelines soon 🙈)

Reply:
Yes we do, some day…